### PR TITLE
fixes #8942 - ability to override tftp root parameter

### PR DESCRIPTION
**Pull request by gpetras**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Manage TFTP
```
Comment (domcleal) [inline manifests/init.pp]: Could you add this parameter to the class documentation please?

**Comment by domcleal** [comment]
:+1: we should add the other params too at some point

File: spec/classes/init_spec.rb [hunk 2]
```diff
@@ -157,6 +157,27 @@
...
 
+  context 'on RedHat with root set to /tftpboot' do
+    let :facts do
+      {
+        :osfamily               => 'Redhat',
+        :operatingsystemrelease => '6.4',
+      }
+    end
+
+    let :pre_condition do
```
Comment (ekohl) [inline spec/classes/init_spec.rb]: This can be done through ```let :params```

**Comment by mmoll** [comment]
merged, thanks @gpetras :tada:

**Comment by ekohl** [comment]
I now realize we could have added a ```validate_absolute_path```, but this is still a good patch in itself.

**Comment by gpetras** [comment]
I can submit another for that if you'd like, @ekohl.

**Comment by ekohl** [comment]
@gpetras if you'd do that and attend FOSDEM or cfgmgmtcamp I'll owe you a beer (or other beverage).

**Pull request merged**